.editorconfig

```diff
@@ -0,0 +1,21 @@
+# See: https://EditorConfig.org
```

Comment:
This file doesn't seem to belong as part of this PR.
We should have common settings, but we shouldn't mix things that are out of scope from a PR

Reply:
👍 I will open a separate PR. This PR required writing a lot of ruby code and not having default settings became tedious.